backend/backend/src/main/java/de/aaaaaaah/velcom/backend/restapi/endpoints/GraphDetailEndpoint.java

```diff
@@ -211,8 +291,8 @@ public String getAuthor() {
 			return author;
 		}
 
-		public long getAuthorDate() {
-			return authorDate;
+		public long getCommitterDate() {
```

Comment:
I think this changes the API, but the spec has not been updated to reflect this.

Reply:
True, I forgot to update that file